src/extension.ts

```diff
@@ -1,16 +1,23 @@
 import { ExtensionContext } from 'vscode';
 
-import { registerURIHandler } from './lib/uri';
+import { getRootPath } from './lib/helpers';
 import { registerCommands } from './lib/commands';
+import { registerEvents } from './lib/events';
+import { registerURIHandler } from './lib/uri';
 import { initializeStatusBar } from './lib/statusBar';
-import { getRootPath } from './lib/helpers';
 import { initializeProviders } from './lib/providers';
 import { initializeConfig } from './lib/auth';
 import { initializeTerminal } from './lib/terminal';
-import { registerEvents } from './lib/events';
+import { initializeTracking, trackEvent } from './lib/tracking';
+import { TRACKED_EVENTS } from './lib/constants';
 
 export const activate = async (context: ExtensionContext) => {
-  console.log('Activating Extension...');
+  await trackEvent(TRACKED_EVENTS.ACTIVATE);
+  console.log(
+    'Activating Extension Version: ',
+    // @ts-ignore TODO - Why is extension not available, when it is?
+    context.extension.packageJSON.version
```

Comment:
It wasn't available until v1.55: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/52117/files#diff-0a5f3d01be8ab95a0ca83baa6915417d7497d254e6f0b508447db7388111a2e4R6082

I think it's worth bumping the supported version to 1.55